addon-test-support/ember-mocha/index.js

```diff
@@ -2,14 +2,27 @@ import describeModule    from 'ember-mocha/describe-module';
 import describeComponent from 'ember-mocha/describe-component';
 import describeModel     from 'ember-mocha/describe-model';
 import setupTestFactory  from 'ember-mocha/setup-test-factory';
-import { it }            from 'mocha';
-import { setResolver }   from '@ember/test-helpers';
+import {
+  it,
+  beforeEach,
+  afterEach
+} from 'mocha';
+import {
+  setResolver,
+  setupApplicationContext,
+  teardownApplicationContext,
+  setupContext,
+  teardownContext,
+  setupRenderingContext,
+  teardownRenderingContext
+} from '@ember/test-helpers';
 import {
   TestModule,
   TestModuleForModel,
   TestModuleForComponent,
   TestModuleForAcceptance
 } from 'ember-test-helpers';
+import { assign } from '@ember/polyfills';
```

Comment:
this is not available yet in Ember 2.0 and 2.4. I would prefer to make this a non-breaking change so if we can work around it by falling back to `merge` that would be great.

Reply:
Yes, I saw that. Will fix it.